src/utils/interface.js

```diff
@@ -110,8 +107,6 @@ export const statusBarContainer = () => {
 
 export const statusBar = () => {
   const statusBar = blessed.box({
-    border: false,
-    bottom: 0,
```

Comment:
removing bottom 0 on both of these may break the app. Did you build and see if it was okay?

Reply:
I see, seems the CI is not reporting any error. I will fix then.

Reply:
Unfortunately I don't have a testing environment set up yet, but I meant user acceptance testing. When you run Check It Out after those changes, It should still function the same. Right now I am suspecting that it won't.

I can pull the code down and test later tonight, too

Reply:
How to run this code at my local machine, Is there any documentation about that ? I try to search it, but I am couldn't find.

Reply:
Not a problem! You can run `node index.js` from the project root to execute the application.

I will make a note and update my documentation for some local development steps. Thanks!

Reply:
You right, I have checked on my local machine. So I localize the common style on variable that can be reused. Please check, hopefully it will solve this issue.